telegram/bot.py

```diff
@@ -889,6 +899,8 @@ def send_animation(self,
                        **kwargs):
         """
         Use this method to send animation files (GIF or H.264/MPEG-4 AVC video without sound).
+        On success, the sent Message is returned. Bots can currently send animation files of
```

Comment:
Please use the "Returns:" thingy for the return type, as with the other functions. Can you change that for

* send_document
* send_sticker
* send_video
* send_video_note
* edit_message_media
* send_audio

And maybe also fix the rendering of the return description for stop_poll?

Reply:
Yea I noticed that inconsistency. Wasn't sure which one to go for till now. Will change in all the other methods too.

> And maybe also fix the rendering of the return description for stop_poll?

Yea I noticed that a few minutes back while working on my pollbot. Will fix.